.github/workflows/review.yml

```diff
@@ -172,15 +174,17 @@ jobs:
 
       - name: run nixpkgs-review ${{ inputs.extra-args }}
         run: |
+          # shellcheck disable=SC2046
           nixpkgs-review -- \
-            pr ${PR_NUMBER} \
+            pr "${PR_NUMBER}" \
             --no-shell \
             --no-headers \
             --print-result \
             --build-args="-L" \
             --pr-json="$PR_JSON" \
             $EXTRA_ARGS \
             || true
+          # shellcheck enable=SC2046
```

Comment:
This seems to have confused shellcheck somehow?

```
SC1072:error:13:1: Expected a command. Fix any mentioned problems and try again [shellcheck]
       >     |
       > 176 |         run: |
       >     |         ^~~~
```

Reply:
Turns out `shellcheck enable` doesn't work like you'd expect. It doesn't re-enable a disabled check. It is only meant to enable optional checks. https://github.com/koalaman/shellcheck/wiki/Directive

I fixed it.